meta-mender-core/recipes-mender/mender-client/mender-client.inc

```diff
@@ -272,23 +267,17 @@ do_install() {
     fi
 
     # install dbus files
-    if grep -q install-dbus ${B}/src/${GO_IMPORT}/Makefile; then
-        if ${@bb.utils.contains('PACKAGECONFIG', 'dbus', 'true', 'false', d)}; then
-            # install the D-Bus policy file
-            oe_runmake \
-                -C ${B}/src/${GO_IMPORT} \
-                V=1 \
-                prefix=${D} \
-                datadir=${datadir} \
-                install-dbus
-
-            # install the D-Bus interface file(s)
-            if [ -f ${B}/src/${GO_IMPORT}/Documentation/io.mender.Authentication1.xml ]; then
-                install -d ${D}${datadir}/dbus-1/interface
-                install -m 0644 ${B}/src/${GO_IMPORT}/Documentation/io.mender.*.xml ${D}${datadir}/dbus-1/interface
-            fi
-        fi
-    fi
+    # install the D-Bus policy file
+    oe_runmake \
+        -C ${B}/src/${GO_IMPORT} \
+        V=1 \
+        prefix=${D} \
+        datadir=${datadir} \
+        install-dbus
+
+    # install the D-Bus interface file(s)
+    install -d ${D}${datadir}/dbus-1/interface
+    install -m 0644 ${B}/src/${GO_IMPORT}/Documentation/io.mender.*.xml ${D}${datadir}/dbus-1/interface
```

Comment:
Question: Should we consider adding this to the `Makefile`?

Reply:
Good idea, I will add it.